features/support/env.rb

```diff
@@ -10,12 +10,14 @@
 require 'cucumber/rails'
 require 'capybara/rails'
 require 'email_spec/cucumber'
+require 'capybara-screenshot/cucumber'
```

Comment:
With the addition of cabybara-screenshot, we should update the [`show me the page` debugging step definition](https://github.com/GSA/search-gov/blob/master/features/step_definitions/debugging_steps.rb#L1) to call `screenshot_and_open_image` instead of `save_and_open_page`. (Also: make sure that works ;) )